jaxrs-spec/src/main/asciidoc/chapters/introduction/_status.adoc

```diff
@@ -28,3 +28,11 @@ The project team seeks feedback from the community on any aspect of this
 specification, please send comments to:
 
 jaxrs-dev@eclipse.org
+
+[[context-injection]]
+==== Support for @Context Injection
+
+As part of an effort to better align with https://jakarta.ee/specifications/cdi/[Jakarta CDI],
```

Comment:
Would it make sense to explicitly mention that people should start using `@Inject`?

Reply:
Something to think about that since the Jakarta project now "owns" `@Inject` (the DI spec), it might be an option to update this annotation as part of Jakarta EE 10 to do support annotating parameters.

Not seeing this is absolutely needed, but something to keep in mind as an option.